src/apexguru/apex-guru-service.ts

```diff
@@ -128,16 +127,19 @@ export function transformStringToRuleResult(fileName: string, jsonString: string
     };
 
 	reports.forEach(parsed => {
-		const encodedClassAfter = parsed.properties.find((prop: ApexGuruProperty) => prop.name === 'code_after')?.value;
+		const encodedCodeBefore = parsed.properties.find((prop: ApexGuruProperty) => prop.name === 'code_before')?.value ?? '';
+		const encodedCodeAfter = parsed.properties.find((prop: ApexGuruProperty) => prop.name === 'code_after')?.value ?? '';
+		const lineNumber = parseInt(parsed.properties.find((prop: ApexGuruProperty) => prop.name === 'line_number')?.value);
 
 		const violation: ApexGuruViolation = {
 			ruleName: parsed.type,
 			message: parsed.value,
 			severity: 1,
 			category: parsed.type, // Replace with actual category if available
-			line: parseInt(parsed.properties.find((prop: ApexGuruProperty) => prop.name === 'line_number')?.value),
+			line: lineNumber,
 			column: 1,
-			suggestedCode: Buffer.from(encodedClassAfter, 'base64').toString('utf8')
+			currentCode: Buffer.from(encodedCodeBefore, 'base64').toString('utf8'),
+			suggestedCode: Buffer.from(encodedCodeAfter, 'base64').toString('utf8'),
```

Comment:
If currentCode and suggestedCode is equal to `` (i.e. empty)… does it show this in the popup box still?
I ask because I would have assumed that to trigger whether it shows up or not would be based on undefined or null instead. Because maybe the after code is indeed empty and we want to show that instead of hide it.

Reply:
As explained in slack, this doesn't show up.

Reply:
Maybe not today. But it could in the future.

Instead of 
```
(prop: ApexGuruProperty) => prop.name === 'code_after')?.value ?? ''
```

Why not just send in the value always even if it is undefined. 

Then your line
```
if (apexGuruViolation.suggestedCode) {
```
code could just switch to
```
if (apexGuruViolation.suggestedCode !== undefined) {
```
so that it allows for empty strings.